specs-blocks/gutenberg-markdown-block-spec.js

```diff
@@ -31,37 +42,33 @@ if ( screenSize !== 'mobile' ) {
 	describe( `[${ host }] Gutenberg Markdown block: (${ screenSize }) @jetpack`, function() {
 		this.timeout( mochaTimeOut );
 
-		describe( 'Publish a simple post with Markdown block', function() {
-			const expectedHTML = `<h3>Header</h3>
-<p>Some <strong>list</strong>:</p>
-<ul>
-<li>item a</li>
-<li>item b</li>
-<li>item c</li>
-</ul>
-`;
-			// Easy way to run/develop tests against local WP instance
-			// step( 'Can login to WPORG site', async function() {
-			// 	const loginPage = await WPAdminLogonPage.Visit( driver, 'http://wpdev.localhost/' );
-			// 	await loginPage.login( 'wordpress', 'wordpress' );
-			// } );
-
-			step( 'Can create wporg site and connect Jetpack', async function() {
+		before( 'Prepare Site for testing', async function() {
```

Comment:
At some point, this `before` block can be transferred into some sort of Gutenberg-prep flow, which will prepare sites for all possible cases: WPCOM-Calypso, WPCOM-wp-admin, Jetpack-Calypso, Jetpack-wp-admin.
This new flow also can include `Can start new post` step.